pyproject.toml

```diff
@@ -19,7 +19,7 @@
 [tool.poetry]
 name = "calamus"
 version = "0.3.0"
-description = "calamus is a library built on top of marshmallow to allow (de-)Serialization of Python classes to JSON-LD."
+description = "calamus is life."
```

Comment:
😆

Reply:
🤣 I clicked the create PR button so fast that didn't realize it's not based on my own fork.